NEWS.md

```diff
@@ -1,3 +1,37 @@
+### 7.0.0 (2020-11-09):
```

Comment:
I would separate out the items for adding Node 14 support and dropping Node 8. Just make easily visible/searchable. Prob something like 'Added official parity support for Node 14' as the first item.

I'd recommend we make the "BREAKING" lines bold.

Finally, with the dropping of Node 8... I'd prob include a link to the supported versions policy (of which we prob need to update for Node 16 now).

```
For further information on our
  support policy, see:
  https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.
```

Reply:
Split out and bolded BREAKING